src/timeseriesflattener/flattener.py

```diff
@@ -151,6 +155,9 @@ def aggregate_timeseries(self, specs: Sequence[ValueSpecification]) -> Aggregate
             dfs, pred_time_uuid_col_name=self.predictiontime_frame.pred_time_uuid_col_name
         )
 
+        if not self.compute_lazily:
```

Comment:
the frame classes coerces dfs back into lazy in the post init method, so had to add it here instead